src/components/scrollable-list/scrollable-item-wrapper.component.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import ScrollableListContext from './scrollable-list.context';
+
+const ScrollableItemWrapper = (CaptiveComponent, index, isSelected) => (
```

Comment:
As this isn't actually a component it might make sense to rename the file and const to something more applicable and less confusing

Reply:
What do you think might be more applicable and less confusing? Something more imperative like `makeSelectable`?